source/administration/config-settings.rst

```diff
@@ -306,35 +306,39 @@ Restrict the permission level required to rename and set the header or purpose f
 
 **All channel members**: Allow all channel members to rename public channels.
 
-**Team and System Admins**: Restrict renaming public channels to Team and System Admins that are members of the channel.
+**Channel Admins, Team Admins, and System Admins**: Restrict renaming public channels to Channel Admins, Team Admins, and System Admins that are members of the channel.
```

Comment:
I think in this case "that" is correct because it refers to the System Admin role, not the System Administrator person.